Style_guide (#1332)
spectrocloud/librarium: styleguide/tutorial-layout.md

# Tutorial Layout

> To create an impactful tutorial guide, you must create a tutorial plan and obtain stakeholder approval for alignment. Adhere to the [Tutorial Plan](#tutorial-plan) section, and the [Tutorial Layout](#tutorial-layout) section to deliver accurate, user-friendly, and impactful tutorials consistently. 

Reviewer: ```suggestion
All Spectro Cloud tutorials must adhere to the following layout. Review the layout guide when structuring a tutorial. Before you bring writing a tutorial, we recommend you go through the exercise of planning out the tutorial. Use the [Tutorial Plan Template](https://github.com/rahulhazra97/Documentation-Guide/wiki/Tutorial-Plan-Template) to get started. Planning out a tutorial and identifying the scope and audience results in a higher quality deliverable and prevents writing content that may not be applicable to the tutorial. 

Reach out to us at spectro-docs@spectrocloud.com to propose a tutorial idea and share the plan with us.
```

We can combine the intro with the following section.

| Clone GitHub Repository   |  Most tutorials are expected to have a public GitHub repository containing a versioned copy of the tutorial code, if applicable. In this section, the user should be walked through the following commands:   |
| [Concept]                 | The title of one of the main concepts of the tutorial. This could be one of several main concepts.  Example - “Configure the Environment” |
| [Sub-Concept]             |  The title of a child concept stems from the parent concept.  Example - “Create Secondary Cluster” OR “App Model  Overview” |
| Cleanup                   |  All tutorials should ideally be deployed through infrastructure as code (IaC) if applicable.  All resources deployed by the tutorial should be removed. It is our responsibility to help and provide guidance to the user on how to remove all resources. Sometimes this section is brief with a simple command `terraform destroy -auto-approve`.  |
| Wrap-Up                   |  This section summarizes the key learning concepts and the actions the practitioner conducted. Additionally, this section should link to or suggest the next set of topics that the practitioner can dive into. |

> For any further information contact : `spectro-docs@spectrocloud.com

Reviewer: ```suggestion
For any further information contact : spectro-docs@spectrocloud.com
```

## Tutorial Plan

All tutorials require a tutorial plan document with stakeholder signoff before authoring. Writing the plan out prevents incorrect deliverables and reduces the chance of time spent inefficiently. The tutorial plan process will help you create a much higher-quality tutorial.
 
* To get started, make a copy of the tutorial plan from the [Reference Template](https://github.com/rahulhazra97/Documentation-Guide/wiki/Reference-Template) page.
* Connect your Google account.  
* Click on **File >> Make a copy**. 
> :information_source: We use Google Docs to collaborate asynchronously. Using a Google document lets us review and quickly provide feedback through comments and suggestions. <br />

 ![image](https://github.com/rahulhazra97/Documentation-Guide/assets/126905240/72e6eca4-992a-45ea-9c80-ab5f3708deaf)

* Next, copy the template to a Google Drive location of your choosing. <br /> <br />
![image](https://github.com/rahulhazra97/Documentation-Guide/assets/126905240/1d2ce48e-ca26-47e5-be2c-4e4774531783)

When you are ready for a review, tag the education team or notify the team by dropping a message in the Slack `education` channel.

### Tutorial Plan Reference

> To get started with your tutorial plan, create a copy of the following template: 

Author: I did create a child page as reference template on line 20. Is that not a child page? 

Rest removed and correct to our email address.

## Tutorial Layout 

> All Spectro Cloud tutorials are expected to have the same look and feel. The tutorial text and voice must adhere to the Spectro Cloud[ style guide](https://github.com/rahulhazra97/Documentation-Guide/wiki/Spectro-Cloud-Style-Guide) for a universal look and feel. The tutorial layout must also feel familiar to the reader; therefore, all tutorials must follow the layout standard described on this page.

Author: ```suggestion
All Spectro Cloud tutorials are expected to have the same look and feel. The tutorial text and voice must adhere to the Spectro Cloud [style guide](https://github.com/rahulhazra97/Documentation-Guide/wiki/Spectro-Cloud-Style-Guide) for a universal look and feel. The tutorial layout must also feel familiar to the reader; therefore, all tutorials must follow the layout standard described on this page.
```

| Clone GitHub Repository   |  Most tutorials are expected to have a public GitHub repository containing a versioned copy of the tutorial code, if applicable. In this section, the user should be walked through the following commands:   |
| [Concept]                 | The title of one of the main concepts of the tutorial. This could be one of several main concepts.  Example - “Configure the Environment” |
| [Sub-Concept]             |  The title of a child concept stems from the parent concept.  Example - “Create Secondary Cluster” OR “App Model  Overview” |
| Cleanup                   |  All tutorials should ideally be deployed through infrastructure as code (IaC) if applicable.  All resources deployed by the tutorial should be removed. It is our responsibility to help and provide guidance to the user on how to remove all resources. Sometimes this section is brief with a simple command `terraform destroy -auto-approve`.  |
| Wrap-Up                   |  This section summarizes the key learning concepts and the actions the practitioner conducted. Additionally, this section should link to or suggest the next set of topics that the practitioner can dive into. |

> For questions or concerns, please ask in the  Slack channel `education`.

Author: ```suggestion
For questions or concerns, please mail to our email address [spectro-docs@spectrocloud.com](spectro-docs@spectrocloud.com).
```

> For questions or concerns, please ask in the  Slack channel `education`.

Author: Done.
